packages/desktop-client/src/components/reports/reports/CustomReport.jsx

```diff
@@ -312,7 +312,23 @@ export function CustomReport() {
 
   return (
     <View style={{ ...styles.page, minWidth: 650, overflow: 'hidden' }}>
-      <Header title="Custom Reports" />
+      <View
+        style={{
+          flexDirection: 'row',
+          flexShrink: 0,
+        }}
+      >
+        <Header title="Custom Reports:" />
+        <Text
+          style={{
+            ...styles.veryLargeText,
+            marginTop: 40,
+            color: theme.pageTextPositive,
+          }}
+        >
+          {report.name ?? 'Unsaved report'}
```

Comment:
```suggestion
          {report.name || 'Unsaved report'}
```

Nullish operator isn't going to work here, because we have an empty string (falsey) rather than nullish.